Monika After Story/game/script-topics.rpy

```diff
@@ -16165,3 +16165,57 @@ label monika_giving_criticism:
     m 3dsu "Graciously accept you can't change everyone's mind and stay considerate when assessing someone else's work."
     m 3hub "...That'd be Monika's Critique Tip of the Day, ahaha!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_curse_words",
+            category=["advice", "life"],
+            prompt="Curse words",
+            random=True,
+            pool=False
+        )
+    )
+
+default persistent._mas_pm_swears_occasionally = None
```

Comment:
```suggestion
default persistent._mas_pm_swear_frequency = 0
```

Reply:
Oh, I didn't think of it, sorry! I'll fix it now.

Reply:
I think it needs to be equated to 0, not, None, as in the value of the "No": category, so the variable is saved as an integer and not a boolean. Not entirely sure though.

Reply:
`None` is not a boolean type. It is its own type, a `Nonetype`

Reply:
We *should* default this to `None` as it shows we have not answered yet.